graph-db/src/utils/cmr/clearScrollSession.js

```diff
@@ -0,0 +1,28 @@
+const fetch = require('node-fetch')
+
+/**
+ * Given a CMR Scroll Session ID, clear that session from
+ * memory, freeing up more CMR resources
+ * @param {String} scrollId
+ * @returns Status code response
+ */
+exports.clearScrollSession = async (scrollId) => {
+  if (!scrollId) {
+    console.warn(`Scroll ID was null or undefined: [${scrollId}]`)
+    return null
```

Comment:
are you specifically looking for `null` or will just `return` to return an `undefined` be ok?

Reply:
The linter complains when I return `undefined`